main.py

```diff
@@ -28,24 +30,33 @@ def delete_table(table_name: str):
     result = TableManager.delete_table(client, table_name)
     if not result:
         raise HTTPException(status_code=400, detail='Table does not exist.')
+    return Response.ok()
 
 
 @app.get("/entities")
 def query_entities(table_name: str):
     client = ClientFactory.getTableClient(table_name)
     result = EntityManger.query_entities(client)
-
+    if not result:
+        raise HTTPException(status_code=500, detail='Cannot create the entity.')
```

Comment:
Confused by the error message. I assume it should be something like `Cannot query entities`.

Reply:
My careless about the detail. It should have been "Cannot query the entities."